dkpro-core-opennlp-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/opennlp/OpenNlpNamedEntityRecognizerTrainer.java

```diff
@@ -95,6 +96,14 @@
     private File targetLocation;
 
     /**
+     * Regex to filter the {@link de.tudarmstadt.ukp.dkpro.core.api.ner.type.NamedEntity#getValue() named entity} by
+     * type.
+     */
+    public static final String PARAM_ACCEPTED_TAGS_REGEX = ComponentParameters.PARAM_ACCEPTED_TAGS_REGEX;
+    @ConfigurationParameter(name = PARAM_ACCEPTED_TAGS_REGEX, mandatory = false)
```

Comment:
This should be mandatory since this parameter cannot be null, right?

Reply:
I added null checks [here](https://github.com/dkpro/dkpro-core/pull/1150/files#diff-3323f5778bcff3b84c5f1e151d1deb08R158) and [here](https://github.com/dkpro/dkpro-core/pull/1150/files#diff-1cd5d8d905ce58f58da8d3b8f4efefe1R99). So it shouldn't be mandatory. 

I didn't want to break backwards compatibility.